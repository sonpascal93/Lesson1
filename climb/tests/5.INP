121
-13 19 0 -7 8 6 1 7 -19 -18 0 0 3 -19 7 0 19 -6 -16 -11 19 -5 1 -15 -15 -4 -19 -5 17 -18 -6 -4 -19 14 -8 -9 -14 14 -5 6 18 -4 -2 14 11 15 -7 9 14 -3 7 1 -17 6 -12 14 -19 12 11 -10 -10 -4 17 -17 14 -16 5 3 -17 -13 -15 -5 10 -1 4 -16 13 7 12 -18 6 6 11 14 -5 -15 4 0 3 -6 5 -15 0 6 -16 -2 15 6 -12 3 10 0 18 -15 -3 5 -8 12 -15 -18 -10 6 -19 5 -8 11 -2 7 -17 -18 5 