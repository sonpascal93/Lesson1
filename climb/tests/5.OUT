29
9 10 11 12 13 15 17 21 29 34 38 44 49 56 58 59 73 78 81 82 87 88 93 96 99 104 109 110 113 